stylus/cozy-ui/build.styl

```diff
@@ -351,32 +351,6 @@
 */
 @require '../../react/Checkbox/styles.styl'
 
-/*
- Input Radio
-
- Markup:
- <form>
-    <div>
-        <label class="u-mb-half c-input-radio {{modifier_class}}">
-            <input type="radio" name="radio"/>
-            <span>I'm a radio one</span>
-        </label>
-        <label class="u-mb-half c-input-radio {{modifier_class}}">
-            <input type="radio" name="radio"/>
-            <span>I'm a radio two</span>
-        </label>
-    </div>
- </form>
-
- .is-error - Error variant
-
- Weight: 4
-
- Styleguide Components.form.input-radio
-*/
-@require '../../react/Radio/styles.styl'
```

Comment:
why don't we delete the Radio/styles.styl file? 

Also this commit is a BC since now the content of `Radio/styles` is not in the final bundle.

Reply:
Pas le temps de gérer l'impact qu'il y a eu dans ActionMenu, on ne va plus faire de BC, je vais revert ça